gojira.sh

```diff
@@ -761,6 +786,9 @@ main() {
              "compatible base, but remember to run 'make dev'!"
     fi
 
+    # Add CLI environment flags to Kong instance
+    [[ ! -z $GOJIRA_CLI_ENVS ]] && add_egg cli_envs
```

Comment:
Why only on the `up` action? This might be relevant to other commands, like any `gojira compose` or `gojira lay`.

Reply:
Good call, forgot about those. I might just put it back inside `p_compose` where I had it before so it is available for all actions even if they don't need it (e.g. `run` as it will have zero effect).